hudi-integ-test/src/test/resources/unit-test-cow-dag.yaml

```diff
@@ -17,23 +17,53 @@ first_insert:
   config:
     record_size: 70000
     num_insert_partitions: 1
-    repeat_count: 1
+    repeat_count: 2
```

Comment:
@n3nash : my bad. can you clarify what does repeat do here. will it add 2* 100 records ?  If yes, total expected records should be 400 right in line 66.

Reply:
We rollback 1 InsertNode, so total 2*100 + 100 - 100 + 100.

Reply:
my bad, I mistook num records in first_upsert as insert records. just now noticed it as upsert records.